README.md

```diff
@@ -1,120 +1,115 @@
+<!-- PROJECT LOGO -->
+<p align="center">
+    <h1 align="center">Dumbarton</h1>
+    <p align="center">
+        A simple Jekyll theme for academics
+        <br />
+        <a href="#table-of-contents"><strong>Explore the docs »</strong></a>
+        <br />
+        <br />
+        <a href="https://dumbarton.netlify.app/">View Demo</a>
+        ·
+        <a href="https://github.com/tcbutler320/Jekyll-Theme-Dumbarton/issues">Report Bug</a>
+        ·
+        <a href="https://github.com/tcbutler320/Jekyll-Theme-Dumbarton/issues">Request Feature</a>
+        </p>
+</p>
+
 [![Contributors][contributors-shield]][contributors-url]
 [![Forks][forks-shield]][forks-url]
 [![Issues][issues-shield]][issues-url]
 [![LinkedIn][linkedin-shield]][linkedin-url]
 [![Netlify Status](https://api.netlify.com/api/v1/badges/24a6b9c4-6586-4e8d-8540-92942a0c47e8/deploy-status)](https://app.netlify.com/sites/admiring-bassi-e955ad/deploys)
-![](https://img.shields.io/badge/Bootstrap-v4.5.3-blue)  
-
-<!-- PROJECT LOGO -->
-<p align="center">
-  <h1 align="center">Dumbarton</h1>
-  <p align="center">
-    A simple Jekyll theme for academics
-    <br />
-    <a href="#table-of-contents"><strong>Explore the docs »</strong></a>
-    <br />
-    <br />
-    <a href="https://dumbarton.netlify.app/">View Demo</a>
-    ·
-    <a href="https://github.com/tcbutler320/Jekyll-Theme-Dumbarton/issues">Report Bug</a>
-    ·
-    <a href="https://github.com/tcbutler320/Jekyll-Theme-Dumbarton/issues">Request Feature</a>
-  </p>
-</p>
+![Bootstrap](https://img.shields.io/badge/Bootstrap-v4.5.3-blue) 
 
-![](https://raw.githubusercontent.com/tcbutler320/Jekyll-Theme-Dumbarton/dev/screenshot.png)
```

Comment:
I typically would not use raw images on the readme, however this was done to satisfy [Jekyll-Themes.com](https://jekyll-themes.com/submit/). For whatever reason, they require links on the readme to be raw links.  

<img width="966" alt="Screen Shot 2020-11-24 at 8 56 20 PM" src="https://user-images.githubusercontent.com/41709108/100172973-83234c00-2e97-11eb-8bf1-278a1921b48d.png">

Reply:
Aha. They must be pulling the README as a preview on their site, so a relative link on GitHub won't work without them doing extra to it.

No problem, I'll adjust